src/kontrol/foundry.py

```diff
@@ -258,12 +258,14 @@ def build(self) -> None:
 
     @cached_property
     def all_tests(self) -> list[str]:
+        test_dir = self.profile.get('test', 'test')
+        test_dir += '/' if not test_dir.endswith('/') else ''
```

Comment:
you could drop the `if` statement in favor of `os.path.join`:

```suggestion
        test_dir = os.path.join(self.profile.get('test', 'test'), '')
```

Reply:
Nice, I didn't know about it, thank you!